src/main/java/io/xpring/xrpl/DefaultXRPClient.java

```diff
@@ -303,7 +307,11 @@ public RawTransactionStatus getRawTransactionStatus(String transactionHash) thro
 
     byte[] transactionHashBytes = Utils.hexStringToByteArray(transactionHash);
     ByteString transactionHashByteString = ByteString.copyFrom(transactionHashBytes);
-    GetTransactionRequest request = GetTransactionRequest.newBuilder().setHash(transactionHashByteString).build();
+    // Note, if ledger_index_min is non-zero and ledger_index_max is 0, the
+    // software will use the max validated ledger in place of ledger_index_max
```

Comment:
I validated with Mark Travis on rippled that we don't ned to set this. If unset, all ledgers are searched. 

You can omit this code.